client/webpack.client.base.config.js

```diff
@@ -59,7 +60,7 @@ module.exports = {
     loaders: [
       { test: /\.(woff2?|svg)$/, loader: 'url?limit=10000' },
       { test: /\.(ttf|eot)$/, loader: 'file' },
-      { test: /\.(jpe?g|png|gif|svg|ico)$/, loader: 'url?limit=10000' },
+      { test: /\.(jpe?g|png|gif|svg|ico)$/, loader: `customfile-loader?limit=10000&name=[hash]_[name].[ext]${assetHost}` },
```

Comment:
Would it make sense, for the sake of consistency, to use the same filename pattern Rails is using? That is, `[name]-[hash].[ext]`. Another benefit is that in this naming schema the file names are sortable.